app/src/main/java/org/mozilla/reference/browser/ext/Context.kt

```diff
@@ -23,3 +28,21 @@ val Context.components: Components
 
 fun Context.getPreferenceKey(@StringRes resourceId: Int): String =
     resources.getString(resourceId)
+
+/**
```

Comment:
nit:
```Kotlin
/**
 * Creates a share intent.
 *
 * @param ...
 * @param ...
 * @return true if intent was ..., otherwise false
 */
```
This might also be a candidate for the a-c repo in the future :)

Reply:
👍